### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maclover7**
Small Readme ordering and grammar changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,13 +97,9 @@ Returns a url reference to the Base64-encoded asset at the specified path.
...
-    $ bundle install
-    $ bundle exec rake test
+```
+$ bundle install
+$ bundle exec rake test
+```
 
 If you need to test against local gems, use Bundler's gem :path option in the Gemfile and also edit `test/support/test_helper.rb` and tell the tests where the gem is checked out.
-
-## Code Status
```
Comment (rafaelfranca) [inline README.md]: Could you change the patch to keep this section here?

Comment (maclover7) [inline README.md]: Moved badges to top of readme

Comment (rafaelfranca) [inline README.md]: Can we keep at bottom?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
 
 This gem provides official integration for Ruby on Rails projects with the Sass stylesheet language.
 
-## Installing
+## Installation
 
 Since Rails 3.1, new Rails projects will be already configured to use Sass. If you are upgrading to Rails 3.1 you will need to add the following to your Gemfile:
 
-    gem 'sass-rails'
+```
```
Comment (robin850) [inline README.md]: This is nit-picky but the real advantage of fenced code blocks is to be able to specitfy the language ; could you add `ruby` after the three back-ticks please ?

File: README.md [hunk 3]
```diff
@@ -31,11 +37,13 @@ can be found on the Sass Website with the following caveats:
 
 ### Example
 
-    MyProject::Application.configure do
-      config.sass.preferred_syntax = :sass
-      config.sass.line_comments = false
-      config.sass.cache = false
-    end
+```
```
Comment (robin850) [inline README.md]: Ditto.

File: README.md (hunk 2 above)
Comment (maclover7) [inline README.md]: :+1:

File: README.md (hunk 3 above)
Comment (maclover7) [inline README.md]: :+1:

**Pull request closed without merging**